### PR TITLE
inject CorruptedDataFilesHandler instead of making it static

### PR DESCRIPTION
**Pull request by christophsturm**
just a small rabbit-hole that i went down while looking at the code. 

storage can probably be injected directly in more places but i didn't want to change any logic in this pr.

### CODE REVIEW

File: core/src/main/java/bisq/core/arbitration/DisputeManager.java [hunk 1]
```diff
@@ -137,7 +138,7 @@ public DisputeManager(P2PService p2PService,
                           OpenOfferManager openOfferManager,
                           KeyRing keyRing,
                           PersistenceProtoResolver persistenceProtoResolver,
-                          @Named(Storage.STORAGE_DIR) File storageDir) {
+                          @Named(Storage.STORAGE_DIR) File storageDir, CorruptedDatabaseFilesHandler corruptedDatabaseFilesHandler) {
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/arbitration/DisputeManager.java]: Can you add a line break. So params are better separated/readable?

**Review by ManfredKarrer — APPROVED** [review]
utACK

File: core/src/main/java/bisq/core/arbitration/DisputeManager.java (hunk 1 above)
Comment (christophsturm) [reply core/src/main/java/bisq/core/arbitration/DisputeManager.java]: sure, i wonder if there is a way to make idea do it automatically.

**Comment by ripcurlx** [comment]
Just a small thing regarding commit messages. Please start them with uppercase https://chris.beams.io/posts/git-commit/

**Review by ripcurlx — APPROVED** [review]
utACK - But please see my comment for future commits.

**Pull request merged**